dist/images/ovnkube.sh

```diff
@@ -678,6 +680,45 @@ ovn-master () {
   exit 9
 }
 
+# v2 v3 - run ovnkube --ha master
+ovn-ha-master () {
```

Comment:
Could we actually just fold this into ovn-master() but use an argument to the function to control the new ovnkube arg? It looks pretty close to ovn-master().

Reply:
There is a difference between ovn-ha-master and ovn-master.

In the case of ovn-ha-master the functionality will not work if --nb-address and --sb-address options are provided. In the case of ovn-ha-master, we want the ovnkube-master to talk to the DB servers using the local unix socket instead of TCP endpoints.

I will leave this AS IS and update the PR. We can discuss further if you have any comments on concerns.

Reply:
@numansiddique Can we have ovnkube --master exit with an error if both --manage-db-servers and --[nb|sb]-address are set? Or perhaps in this case --manage-db-servers takes precedence, a warning is printed, and --[nb|sb]-address are blanked out?

Would that allow combining the bash functions?

Reply:
Done. I have now removed this new function - ovn-ha-master.